<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.0.2 (Python 3.11.2 on win32)" generated="20230420 03:57:31.632" rpa="false" schemaversion="3">
<suite id="s1" name="DDT-XLS" source="D:\TechProEdu\PycharmProjects\Automation_RobotFramework\Tests\DataDrivenTesting\DDT-XLS.robot">
<kw name="Start TestCase" library="Common" type="SETUP">
<kw name="Open Browser" library="SeleniumLibrary">
<arg>${url}</arg>
<arg>${browser}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20230420 03:57:31.777" level="INFO">Opening browser 'chrome' to base url 'https://www.saucedemo.com/'.</msg>
<status status="PASS" starttime="20230420 03:57:31.777" endtime="20230420 03:57:34.966"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="PASS" starttime="20230420 03:57:34.967" endtime="20230420 03:57:35.085"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2s</arg>
<doc>Pauses the test executed for the given time.</doc>
<msg timestamp="20230420 03:57:37.085" level="INFO">Slept 2 seconds</msg>
<status status="PASS" starttime="20230420 03:57:35.085" endtime="20230420 03:57:37.085"/>
</kw>
<status status="PASS" starttime="20230420 03:57:31.776" endtime="20230420 03:57:37.085"/>
</kw>
<test id="s1-t1" name="Verify Login Fails Invalid Creds" line="12">
<kw name="Invalid Login Scenarios">
<arg>${username}</arg>
<arg>${password}</arg>
<arg>${error_msg}</arg>
<msg timestamp="20230420 03:57:37.086" level="FAIL">Variable '${username}' not found.</msg>
<status status="FAIL" starttime="20230420 03:57:37.086" endtime="20230420 03:57:37.086"/>
</kw>
<status status="FAIL" starttime="20230420 03:57:37.085" endtime="20230420 03:57:37.087">Variable '${username}' not found.</status>
</test>
<kw name="Finish TestCase" library="Common" type="TEARDOWN">
<kw name="Close Browser" library="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<status status="PASS" starttime="20230420 03:57:37.088" endtime="20230420 03:57:39.329"/>
</kw>
<status status="PASS" starttime="20230420 03:57:37.088" endtime="20230420 03:57:39.329"/>
</kw>
<status status="FAIL" starttime="20230420 03:57:31.633" endtime="20230420 03:57:39.329"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="DDT-XLS">DDT-XLS</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230420 03:57:31.771" level="ERROR">[ DataDriver ] Error in robot file:
  File "D:\TechProEdu\PycharmProjects\Automation_RobotFramework\Tests\DataDrivenTesting\DDT-XLS.robot", line 0</msg>
<msg timestamp="20230420 03:57:31.772" level="ERROR">In source file:
  File "D:\TechProEdu\PycharmProjects\Automation_RobotFramework\Tests\DataDrivenTesting\../TestData/TestData.xlsx", line 0</msg>
<msg timestamp="20230420 03:57:31.775" level="ERROR">Calling method '_start_suite' of listener 'DataDriver' failed: ImportError: Requirements (pandas, openpyxl) for XLSX support are not installed.
    Use 'pip install -U robotframework-datadriver[XLS]' to install XLSX support.</msg>
</errors>
</robot>
